nemo/collections/nlp/data/glue_benchmark/glue_benchmark_dataset.py

```diff
@@ -121,7 +121,7 @@ def __init__(
 
         self.examples = processor.get_dev_examples(data_dir) if evaluate else processor.get_train_examples(data_dir)
         processor_name = type(processor).__name__
-        tokenizer_type = type(tokenizer).__name__
+        tokenizer_type = type(tokenizer.tokenizer).__name__
```

Comment:
Recommend not to use the tokenizer.tokenizer directly from our code other than from the wrapper itself. Our wrapper is supposed to encapsulate the tokenizer. Such encapsulation enables us to have control and later changes and updates easier. Is there a way to set the tokenizer.__name__ to the value of tokenizer.tokenizer.__name__ somehow in the initialization?

Reply:
done